tests/cfgs/default/result/zoom.pcap.out

```diff
@@ -43,7 +43,7 @@ JA3 Host Stats:
 	1	 192.168.1.117            	 4      
 
 
-	1	UDP 192.168.1.117:58327 <-> 109.94.160.99:8801 [proto: 189/Zoom][IP: 0/Unknown][Encrypted][Confidence: DPI][DPI packets: 1][cat: Video/26][10 pkts/7806 bytes <-> 175 pkts/184434 bytes][Goodput ratio: 95/96][1.44 sec][bytes ratio: -0.919 (Download)][IAT c2s/s2c min/avg/max/stddev: 0/0 14/8 32/35 11/5][Pkt Len c2s/s2c min/avg/max/stddev: 55/60 781/1054 1071/1071 444/129][PLAIN TEXT (replace)][Plen Bins: 1,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,97,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]
```

Comment:
I don't think Zoom is cleartext

Reply:
It is not. Somehow while "Zoom" is correctly associated to "Encrypted", "RTP.Zoom" is associated to "ClearText", which seems wrong to me. Probably an unrelated bug; I'll try to fix in a new version of this PR

Reply:
@lucaderi, what do you think of https://github.com/ntop/nDPI/pull/1955?